.circleci/config.yml

```diff
@@ -1269,10 +1269,34 @@ workflows:
           context: consul-ci
           requires:
             - dev-upload-docker
+      - acceptance-gke-1-23:
```

Comment:
Will remove before merge?

Reply:
haha! yup! i had that comment written down but forgot to hit publish 😂